scipy/spatial/_plotutils.py

```diff
@@ -61,6 +61,22 @@ def delaunay_plot_2d(tri, ax=None):
     -----
     Requires Matplotlib.
 
+    Examples
+    --------
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.spatial import Delaunay, delaunay_plot_2d
+
+    The Delaunay triangulation of a set of random points:
+
+    >>> points = np.random.rand(30, 2)
+    >>> tri = Delaunay(points)
+
+    Plot it:
+
+    >>> delaunay_plot_2d(tri)
+    >>> plt.show()
```

Comment:
Since this code is executed as part of our `doctest` / refguide check suite, and matplotlib produces the output `<Figure size 640x480 with 1 Axes>`, the doc testing fails.

One solution is to add `<Figure size 640x480 with 1 Axes>` below `plt.show()` here and in the cases below. Alternatively, we could relax the checking on `plt.show()`, but that's probably a trickier fix short-term.

Reply:
Perhaps slightly curious that we do have other cases where `plt.show()` works just fine in docs though.

Reply:
The problem is that previous command should return None but returns a figure. maybe assigning the fig to a variable can silence it?

Reply:
Oh right, yeah maybe `_ = delaunay_plot_2d(tri)` to supress